rs/embedders/fuzz/src/imports.rs

```diff
@@ -0,0 +1,94 @@
+// https://internetcomputer.org/docs/current/references/ic-interface-spec#system-api-imports
+pub(crate) const SYSTEM_API_IMPORTS: &str = r#"
+(module
+  (type (;0;) (func))
+  (type (;1;) (func (result i32)))
```

Comment:
The `wasmtime::Linker` has a method [`iter`](https://docs.rs/wasmtime/latest/wasmtime/struct.Linker.html#method.iter) which lists all the functions defined in the linker. Maybe we should have a test that everything in the system api linker appears here so that we don't need to remember to update it? Or maybe it's even possible to generate this module from that information in the linker?
Of course we would need to have exceptions for the "hidden" APIs.

We can also do this as a separate follow-up if it's a big mess.

Reply:
I would definitely like to add this since I don't want this file going out of sync with imports or rather have a single source of truth. I will do it in a follow up, as currently the fuzzers are broken on master